File: cap-1/javascript/pesquisa-simples.js
function PesquisaSimples(lista, item){
    for (let i = 0; i < lista.length; i++) {
        if (lista[i]===item) {
            return i;
        }
    }
    return undefined;
}

minha_lista = Array.from({length: 1000000}, (_, i)=> i+ 1);
console.log(PesquisaSimples(minha_lista, 756000))
console.log(PesquisaSimples(minha_lista, -1))

File: cap-1/python/pesquisa-simples.py
def pesquisa_simples(lista, item):
    for i in range(len(lista)):
        if lista[i] == item:
            return i
    return None


minha_lista = list(range(1, 1000001))

print(pesquisa_simples(minha_lista, 756000))
print(pesquisa_simples(minha_lista, -1))

